woocommerce/woocommerce-android
# Use gradle `configure` plugin

This PR uses the gradle `configure` plugin instead of `bundler`, which should remove the need for a ruby toolchain for day-to-day development of WCAndroid.

**To Test:**
- Ensure all CI things pass
- Check out the repo from scratch, then switch to this branch. Run `./gradlew applyConfiguration`. Note that it fails with the error "Build configuration file gradle.properties doesn't exist, follow README instructions". Copy `gradle.properties-example` to `gradle.properties`, then run `./gradlew applyConfiguration`. This should replace the default values with the proper ones.

Note: @oguzkocer and I discussed having a more automatic solution that doesn't require duplicating the template file, but there are a lot of technical hurdles to overcome there, so for now this is likely the simplest solution for both OSS and internal contributors.

Update release notes:
